chain.schema.json

```diff
@@ -3,6 +3,7 @@
     "$schema": "https://json-schema.org/draft-07/schema",
     "title": "Cosmos Chain",
     "updatelink": "string",
+    "website": "string",
```

Comment:
should I assume that you meant to add 'website' under properties? I don't think you meant to have it up here...

Reply:
Whoops, thank you. It's been a while since I've messed with json schema tbh.